codegen/service.go

```diff
@@ -221,9 +221,10 @@ func (ms *ModuleSpec) SetDownstream(
 		clientMethod = e.ClientMethod
 
 		// TODO: move generated middlewares out of zanzibar
-		headersPropagate = e.HeadersPropagate
-		reqTransforms    = e.ReqTransforms
-		respTransforms   = e.RespTransforms
+		headersPropagate  = e.HeadersPropagate
+		reqTransforms     = e.ReqTransforms
+		respTransforms    = e.RespTransforms
+		reqrespTransforms = e.DummyReqTransforms
```

Comment:
rename the variable reqrespTransforms is confusing in the context of the previous two lines.

Reply:
Updated it.